doc-Managing_Infrastructure_and_Inventory/topics/Deleting_EBS_Volumes.adoc

```diff
@@ -1,11 +1,12 @@
-[[deleting_cinder_volumes]]
+[[deleting_ebs_volumes]]
 = Deleting a Volume
```

Comment:
Ah, totally understandable. No worries, let's not break anything unneccesary then :)